server/app/models/grid_service.rb

```diff
@@ -67,8 +67,12 @@ def to_path
   end
 
   # @return [String]
-  def name_with_stack
-    "#{self.stack.name}-#{self.name}"
+  def name_with_stack(ignore_default_stack: false, stack_separator: '-')
```

Comment:
Did we come to some conclusion on this? AFAIK two options:

I'm okay with the parametrized `name_with_stack` in the understanding that we'll drop the `null-` prefix from stackless services, so this would become just `grid_service.name_with_stack('/')`.

Alternatively we figure out a name for this new `KONTENA_SERVICE=stack/service` syntax that's used in both the CLI and the kontena service env, and create a new method using that name.